nebula-logger/core/main/configuration/classes/LoggerParameter.cls

```diff
@@ -10,8 +10,10 @@
  */
 @SuppressWarnings('PMD.CyclomaticComplexity, PMD.ExcessivePublicCount, PMD.PropertyNamingConventions')
 public class LoggerParameter {
-    private static Set<String> parametersToLoadDuringTests = new Set<String>();
-    private static Map<String, SObject> mockParameterByDeveloperName = new Map<String, SObject>();
+    // During tests, always load this CMDT record
+    // so that tests use the same format when calling System.debug()
+    private static final Set<String> PARAMETERS_TO_LOAD_DURING_TESTS = new Set<String>{ 'SystemDebugMessageFormat' };
```

Comment:
if you reverse the order of these variables, you can use `SYSTEM_DEBG_MESSAGE_FORMAT` in `PARAMETERS_TO_LOAD_DURING_TESTS`

Reply:
Yup, that makes sense, and I'm disappointed in my past-self for double-harcoding a string 😨 . I'll flip those lines around & use the constant in the list.

Reply:
We've all done it